### ISSUE TITLE
[Request] Conda installable

### ISSUE THREAD

**Issue by jolespin** [Benjamin-Lee/vdsearch#8]
Are there plans to throw this on to Bioconda or your own conda channel?  

Could be helpful for integrating this into existing pipelines.  For example, I'm working on github.com/jolespin/veba and it would be useful to incorporate this workflow into the viral binning pipeline.  Due to all of the dependencies for other packages, I'm only using packages installed through conda (potentially pip but conda is preferred).

**Comment by Benjamin-Lee** [comment]
Yes, this is the plan. We’re working on packaging our circular RNA detection suite separately since it will be far easier to distribute this package as a pure Python project (plus it will be way more performant since it’s now multithreaded). In the meantime, I’m happy to run any data you’re willing to share—just shoot me an email. We’ll prioritize getting a package out ASAP. On Feb 2, 2023, at 7:32 PM, Josh L. Espinoza ***@***.***> wrote:﻿
Are there plans to throw this on to Bioconda or your own conda channel?
Could be helpful for integrating this into existing pipelines.  For example, I'm working on github.com/jolespin/veba and it would be useful to incorporate this workflow into the viral binning pipeline.  Due to all of the dependencies for other packages, I'm only using packages installed through conda (potentially pip but conda is preferred).

**Comment by Benjamin-Lee** [comment]
By the way, I've now added a Dockerfile and published the image to Docker Hub, so this should help ease the install pains in the mean time. The updated instructions are on the README.
